app/src/main/res/layout-land/onboarding_slide.xml

```diff
@@ -46,24 +47,22 @@
         android:gravity="center_horizontal"
         android:text="@{viewModel.title}"
         android:textColor="@color/oppiaPrimaryText"
-        android:textSize="24sp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="1.0"
         app:layout_constraintStart_toEndOf="@+id/slide_image_view"
         app:layout_constraintTop_toTopOf="parent" />
 
       <TextView
         android:id="@+id/slide_description_text_view"
+        style="@style/Subtitle1"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
```

Comment:
```
android:layout_width="0dp"
android:layout_height="wrap_content"
```

The above lines can be removed and you can introduce `horizontalBias`.

Considering that the Heading1 already have these attributes.

Reply:
I have also implemented the suggested changes.